.travis.yml

```diff
@@ -5,6 +5,8 @@ python:
   - "3.7" # current default Python on Travis CI
 install:
   - pip install -e . # install deepreg package with corresponding requirements
+  - pip install pytest-cov
```

Comment:
other dependencies are described in setup.py, maybe we can add these packages there?

Reply:
This isn't strictly a dependency for dev/usage, it's just for code coverage on repo. Still want me to add?

Reply:
ah ok lol but pytest-cov is already in setup

Reply:
:facepalm: fixing now